yes + boop

jbanate opened the pull request:
this all seems about right
- Added a cap to how fast crep can send messages at half a second
- Crep can now parse fractions of a second (!p ccrep 5 1.5)
- Copied the entirety of query and made !p test
- made interfaces.json shorter although idk if that would cause any issues

new commit goys

Added !p limbo
Fixed !p crep
Made !p crep and !p rep more user-friendly by telling the user how to use it if used incorrectly
Added reason arg for the following commands:
!p ban
!p close (Not functioning yet)
!p disband
!p kick
!p mute
!p drain
To use this, just put the reason after the command, for example: !p ban jbanate spamming
If no reason is provided, it will default to "No reason provided."
This will not show in-game (yet, idk whether to add it) and only logs in Discord

- added minMsgDelay variable and made most command delays use them
- re-added the check for a value less than 500 ms in !p crep because i managed to remove it by accident somewhehre

remove testing variable from !p cmd
!p close works now
added a 9s limit to crep duration

A PHAS PLEASE APPROV!!

Conutik commented on src/utils/Interfaces.mjs:
```diff
@@ -35,14 +35,11 @@ export const MessageType = Object.freeze({
   Admin: 4,
-  Owner: 5,
-  BotAccount: 5,
+  Owner, BotAccount: 5,
```
You can't do this in js, it'll give `Owner` as undefined

Conutik commented on src/utils/Interfaces.mjs:
```diff
@@ -35,14 +35,11 @@ export const MessageType = Object.freeze({
-  Splasher: 2,
-  Trusted: 2,
+  HoB, Famous: 1,
+  Splasher, Trusted: 2,
```
Same here

jbanate commented:
changed also new commit:

manually select playernames.json because help me
same with generalDatabase.json and package-lock.json

Added !p limbo
Fixed !p crep
Made !p crep and !p rep more user-friendly by telling the user how to use it if used incorrectly
Added reason arg for the following commands:
!p ban
!p close (Not functioning yet)
!p disband
!p kick
!p mute
!p drain
To use this, just put the reason after the command, for example: !p ban jbanate spamming
If no reason is provided, it will default to "No reason provided."
This will not show in-game (yet, idk whether to add it) and only logs in Discord

Conutik commented on src/mineflayer/commands/party/Test.mjs:
Why change from `misc` to `party` 😭

Conutik commented on src/mineflayer/commands/party/Unban.mjs:
Are you sure that the bot would be spawned in the lobby before running other commands?

Conutik commented on src/mineflayer/commands/party/Ban.mjs:
Same concern as unban
